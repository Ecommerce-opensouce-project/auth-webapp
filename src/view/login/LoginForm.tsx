import * as React from "react";

export default ()=>{
    return <div>
        <div className="login-name">Name</div>
        <div className="login-password">Password</div>
    </div>
};