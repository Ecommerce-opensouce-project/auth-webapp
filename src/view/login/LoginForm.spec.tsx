import * as React from "react";
import { shallow } from "enzyme";
import * as renderer from "react-test-renderer";
import LoginForm from "./LoginForm";

const wrapper = shallow(<LoginForm/>);

describe("Test Login form", ()=>{
    it("match with snapshot", ()=>{
        const tree = renderer.create(<LoginForm/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
    it("renders loginform", ()=>{
        const nameField = wrapper.find('.login-name');
        const passField = wrapper.find('.login-password');
        expect(nameField.exists()).toBe(true);
        expect(passField.exists()).toBe(true);
    })
});
