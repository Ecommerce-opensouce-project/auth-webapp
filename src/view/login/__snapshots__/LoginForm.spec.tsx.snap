// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Login form match with snapshot 1`] = `
<div>
  <div
    className="login-name"
  >
    Name
  </div>
  <div
    className="login-password"
  >
    Password
  </div>
</div>
`;
